File: sample/scraping/python/001.py
# coding: UTF-8
import urllib2
from bs4 import BeautifulSoup

# アクセスするURL
url = "https://www.google.co.jp/"
url = "https://ja.wikipedia.org/wiki/Linux"

html = urllib2.urlopen(url)

soup = BeautifulSoup(html, "html.parser")

title_tag = soup.title

title = title_tag.string

print title_tag

print title

File: sample/scraping/go/001.go
package main
// https://qiita.com/ryurock/items/b0466ad144f5e0555a95

import (
    //"net/http"
    //"io/ioutil"
    "fmt"
    "github.com/PuerkitoBio/goquery"
)


func main() {
    doc, err := goquery.NewDocument("https://ja.wikipedia.org/wiki/Linux")
    if err != nil {
        panic(err)
    }
    doc.Find("title").Each(func(_ int, s *goquery.Selection) {
        fmt.Println(s.Text())
        //url, _ := s.Attr("href")
        //fmt.Println(url)
    })
}
